javascript/code-challenges/challenges-01.test.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named `addOne` that takes an array of numbers, and returns a new array of the numbers, incremented by 1.
+
+Use `forEach` to loop over the input array and work with each value.  Push the new value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  const localArray = [];
+  arr.forEach((number) => {
+    localArray.push(number + 1);
+  });
+  return localArray;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named `addExclamation` that takes an array of strings, and returns a new array of the same strings with an "!" added to the end.
+
+Use `forEach` to loop over the input array. Modify each string, and add the updated value into a local array. Return the local array;
+------------------------------------------------------------------------------------------------ */
+
+const addExclamation = (arr) => {
+  // Solution code here...
+};
+
```

Comment:
Nice work so far!

I wanted to help with this one:
We know that our input is an arr of strings.
Our output needs to be a new array of these same strings, but with an ! at the end.

- Perhaps we can declare a new variable and set it to an empty array. 
- Then we can use forEach to iterate over every element(string) in the arr. 
- For each string, we will add the ! to the string and push it into the new arr that we have defined. 

I hope this helps!